ts/components/ExpiredSessionBanner.tsx

```diff
@@ -0,0 +1,22 @@
+import { Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+
+import I18n from "../i18n";
+import variables from "../theme/variables";
+
+const styles = StyleSheet.create({
+  content: {
+    backgroundColor: variables.brandPrimary,
+    padding: variables.contentPadding
+  }
+});
+
+export const ExpiredSessionBanner: React.SFC<{}> = () => (
```

Comment:
you can as well call it `AlertBanner` and pass the message via props

Reply:
I thought about creating a general banner component, but I didn't know if there will be other banners like this. Ok, I'll do it.